src/Components/CreateImageWizard/CreateImageWizard.scss

```diff
@@ -97,5 +103,16 @@ div.pf-v5-c-alert.pf-m-inline.pf-m-plain.pf-m-warning {
     // flex: 1 1 0
     
     // TO DO: Remove after the fix
-    min-height: 800px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    min-height: 800px;  /* Always go with pixels */
+}
+
+.pf-v5-c-wizard__footer {
+    position: sticky;
+    bottom: 0;
+    padding: 1rem;
+    background-color: white;  // Ensure background covers content below
+    z-index: 10;  // Ensure the footer is above the content
```

Comment:
This is 100 by default:
according to [code](https://github.com/patternfly/patternfly/blob/7d6a2437f14461c32222e6a63ffdca1b122cf0ca/src/patternfly/components/Wizard/wizard.scss#L169) it's using `--pf-v5-global--ZIndex--xs` which has value `100` according to [docs](https://v5-archive.patternfly.org/developer-resources/global-css-variables/#component-variables)

Why are you overriding it to 10, which is less then default?

Reply:
I set z-index: 10 instead of the default 100 to ensure the footer stays above the content without conflicting with other UI elements that might have higher z-index values. I tested it with the default z-index: of 100, but it didn’t give us the expected result.

Reply:
Isn't higher z-index more on top?